examples/embeddable/main.tx

```diff
@@ -0,0 +1,16 @@
+addon "evm" {
+    chain_id = input.chain_id
+    rpc_api_url = input.rpc_api_url
+}
+
+signer "deployer" "evm::secret_key" {
+    secret_key = input.secret_key
+}
+
+action "deploy1" "evm::deploy_contract" {
+    contract = {
```

Comment:
We still need to properly store some of the evaluated inputs from the originally publishing runbook, which will allow  the results of something like `evm::get_contract_from_foundry_project("SimpleStorage")` to be used when publishing the runbook, and us publishing just the correct stuff.

Tracked in the planner [here](https://teams.microsoft.com/l/entity/com.microsoft.teamspace.tab.planner/mytasks?tenantId=0f1e1f86-ff36-4233-ba7c-e2d48a07bb41&webUrl=https%3A%2F%2Ftasks.teams.microsoft.com%2Fteamsui%2FpersonalApp%2Falltasklists&context=%7B%22subEntityId%22%3A%22%2Fv1%2Fpremiumplan%2Fc7a7d962-0836-4ed0-8a67-774535750432%2Forg%2Fdeb736b0-d91b-ef11-9f85-000d3a537223%2Fview%2Fboard%2Ftask%2FF8429AC7-75AC-4EFC-BAEB-26A3B49BDCC7%22%7D)